### DISCUSSION TITLE
Photino with React AND Typescript

### DISCUSSION THREAD

**Discussion by jsimmonstx** [tryphotino/photino.NET#219] [Q&A]
Windows 10, VS2022...

The current project template does not appear to enable typescript. I thought React required typescript, but the project template doesn't generate tsx files (yes, I have the typescript extension installed). I've been a dev for over 40 years, but I'm pretty new to react/javascript dev. 

I've been googling for three days, and there info/help on Photino is woefully scarce.

**Comment by MikeYeager** [comment]
@jsimmonstx You can add TypeScript to an existing project with these instructions https://react.dev/learn/typescript. Alternately, you can follow the Installation Guide on the React web site and copy/paste the resulting app into the Photino.React sample. React supports both JavaScript and TypeScript and we've tried to keep our samples as simple as possible. Please re-open if that doesn't work out for some reason.
